x-pack/solutions/observability/packages/kbn-evals-suite-obs-ai-assistant/evals/utils/knowledge_base.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const testDocs = [
+  {
+    id: 'acme_teams',
+    title: 'ACME DevOps Team Structure',
```

Comment:
@viduni94 I suggested in https://github.com/elastic/kibana/pull/234342#discussion_r2332828494 to add 10+ entries.